fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -49,12 +49,9 @@
     protected abstract Session session();
 
     protected IdentifierConverter<Resource, FedoraResource> translator() {
-        if (idTranslator == null) {
-            idTranslator = new HttpResourceConverter(session(),
-                    uriInfo.getBaseUriBuilder().clone().path(FedoraLdp.class));
-        }
-
-        return idTranslator;
+        return idTranslator == null ? idTranslator =
```

Comment:
For the reasons above (and the fact that no logic is being changed here), please revert this update.